src/test/scala/chiselTests/WireSpec.scala

```diff
@@ -17,4 +18,19 @@ class WireSpec extends ChiselFlatSpec {
   it should "not allow init argument to affect type inference" in {
     assertDoesNotCompile("val x: UInt = WireDefault(UInt(4.W), 2.S)")
   }
+  it should "have source locator information on wires" in {
+    class Dummy extends chisel3.Module {
+      val in = IO(Input(Bool()))
+      val out = IO(Output(Bool()))
+
+      val wire = WireInit(Bool(), true.B)
+      val wire2 = Wire(Bool())
+      wire2 := in
+      out := in & wire & wire2
+    }
+
+    val chirrtl = ChiselStage.emitCHIRRTL(new Dummy)
+    chirrtl should include("wire wire : UInt<1> @[WireSpec.scala 26:26]")
+    chirrtl should include("wire wire2 : UInt<1> @[WireSpec.scala 27:23]")
```

Comment:
I would make these tests slightly more robust, don't match on the actual line numbers.
```suggestion
    chirrtl should include("wire wire : UInt<1> @[WireSpec.scala")
    chirrtl should include("wire wire2 : UInt<1> @[WireSpec.scala")
```